README.md

```diff
@@ -197,6 +198,7 @@ If you are in academia and do not know where to register your account in the fed
 - [baudigital.social](https://baudigital.social/) (Architecture, civil engineering, or urban studies; run by FID BAUdigital)
 - [Biologists.social](https://biologists.social/) (The Company of Biologists community server: cell biology, developmental biology and experimental biology)
 - [Climatescienc.es](https://climatescienc.es/explore) (Climate science and related fields)
+- [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) - (For mental health professionals and the general public to exchange ideas, resources and expertise including but not limited to ethics, politics, practice building, and clinical issues; See [here](https://www.clinicians-exchange.org/) for more info.)
```

Comment:
```suggestion
- [Clinicians Exchange](https://mastodon.clinicians-exchange.org/) (For mental health professionals and the general public to exchange ideas, resources and expertise including, but not limited to, ethics, politics, practice building, and clinical issues; [more information](https://www.clinicians-exchange.org/))
```

Reply:
Did you now take the suggestion, or only in parts, or not at all? GH is confusing me here …